editor/components/block-list/style.scss

```diff
@@ -1,3 +1,71 @@
+.components-draggable__clone {
+	.editor-block-list__block-draggable {
+		background: white;
+		box-shadow: $shadow-popover;
+
+		@include break-small {
+			left: 0;
+			right: 0;
+		}
+	}
+
+	// Hide the Block UI when dragging the block
+	// This ensures the page scroll properly (no sticky elements)
+	.editor-block-contextual-toolbar,
+	.editor-block-mover,
+	.editor-block-settings-menu {
+		// I think important is fine here to avoid over complexing the selector
+		display: none !important;
+	}
+}
+
+.editor-block-list__block-draggable {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	z-index: z-index( '.editor-block-list__block-drag-inset' );
```

Comment:
Why do we need this? What happens if it's not here?

Reply:
We want the drag inset to have a lower initial index than anything else positioned *absolutely* within the block to not interfere or be visible in any way. Then we want the drag inset to have a higher index than everything else to cover the area with the grey background when the block is being dragged.

Reply:
>We want the drag inset to have a lower initial index than anything else positioned absolutely within the block to not interfere or be visible in any way.

Is that not already the case if it's absolutely positioned and the first child element?

>Then we want the drag inset to have a higher index than everything else to cover the area with the grey background when the block clone is being dragged.

Isn't this already achieved by `.is-hidden` applying `visibility: none`?

Reply:
> Is that not already the case if it's absolutely positioned and the first child element?

If we can guarantee that it will always be the first child element.

> Isn't this already achieved by .is-hidden applying visibility: none?

That covers everything inside `editor-block-list__block`. If that's enough, then yes.